medicines/web/components/search-results/pagination.tsx

```diff
@@ -1,8 +1,9 @@
 import Link from 'next/link';
 import React from 'react';
 import styled from 'styled-components';
-import { mhraBlue } from '../../styles/colors';
+import { mhra70, mhraBlue, mhrawhite, primaryColor } from '../../styles/colors';
```

Comment:
Should be `mhraWhite` not `mhrawhite`. We're using camel casing on everything else.

Reply:
Done